testing/endtoend/components/eth1/transactions.go

```diff
@@ -91,15 +106,13 @@ func (s *TransactionGenerator) Started() <-chan struct{} {
 	return s.started
 }
 
-func SendTransaction(client *rpc.Client, key *ecdsa.PrivateKey, f *filler.Filler, gasPrice *big.Int, addr string, N uint64, al bool) error {
-	backend := ethclient.NewClient(client)
-
+func SendTransaction(client *rpc.Client, key *ecdsa.PrivateKey, f *filler.Filler, gasPrice *big.Int, addr string, N uint64, backend *ethclient.Client, al bool) error {
 	sender := common.HexToAddress(addr)
-	chainid, err := backend.ChainID(context.Background())
+	nonce, err := backend.PendingNonceAt(context.Background(), fundedAccount.Address)
```

Comment:
why did this change from sender to fundedAccount?

Reply:
Blob transactions and normal transactions can't come from the same address as geth would fail then